### DISCUSSION TITLE
服务器炸了吗？绳网里面的文章都没了.....

### DISCUSSION THREAD

**Discussion by HarmoPharaoh** [share121/inter-knot#4885] [其他]
求回答，跨域助手也没装

**Comment by Dave-12138** [comment]
确定是从[inot.top](https://inot.top/)获取的吗
现在不需要跨域助手，但[网页版](https://www.inot.top/)还是没有反向代理所以只能[装这个浏览器插件](https://microsoftedge.microsoft.com/addons/detail/allow-cors-accesscontro/bhjepjpgngghppolkjdhckmnfphffdag?hl=zh-CN&gl=CN)凑合一下<img src="https://bbs-static.miyoushe.com/static/2024/06/19/f7d6b0099c022d557d12c6662b132406_7125757908673767353.png" alt="流萤 也挺好" width="75" >
或者你想问有些老贴消失的事情： #4569

**Comment by HarmoPharaoh** [comment]
我用了最新的app,但是一直加载不出来(我这里可以访问github和youtube)，换网页版加载一会才出来几个帖子，这是怎么整得

**Reply by Dave-12138** [reply]
因为你连接api.github.com的速度慢、或者公共api token访问频繁所以你需要在绳网里登录（就是授权app）
